docs/dev/contributing.md

```diff
@@ -58,22 +58,110 @@ happier!
 
 ### Reno
 
-To install reno: `pip install reno`
-
 We use `Reno` to create our CHANGELOG. Reno is a pretty simple
-[tool](https://docs.openstack.org/reno/latest/user/usage.html).
+[tool](https://docs.openstack.org/reno/latest/user/usage.html). With each PR
+should come a new releasenotes created with `reno` (unless your change doesn't
+have a single user impact and should not be mentioned in the CHANGELOG, very
+unlikely !).
+
+To install reno: `pip install reno`
 
 Ultra quick `Reno` HOWTO:
 
 ```bash
 $> reno new <topic-of-my-pr> --edit
 [...]
-# Remove unused section and fill the relevant ones
-# Reno will create a new file in releasenotes/notes
+# Remove unused sections and fill the relevant ones.
+# Reno will create a new file in releasenotes/notes.
+#
+# Each section from every releasenote are combined when the CHANGELOG.rst is
+# rendered. So the text needs to be worded so that it does not depend on any
+# information only available in another section. This may mean repeating some
+# details, but each section must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
 [...]
 ```
 
-Then just add and commit the new releasenote (located in `releasenotes/notes/`) with your PR.
+Then just add and commit the new releasenote (located in `releasenotes/notes/`)
+with your PR.
+
+#### Reno sections
+
+The main thing to keep in mind is that the CHANGELOG is written for the agent's
+users and not its developers.
+
+- `features`: describe shortly what your feature does.
+
+  example:
+  ```yaml
+  features:
+    - |
+      Introducing the Datadog Process Agent for Windows.
+  ```
+
+- `enhancements`: describe enhancements here: new behavior that are too small
+  to be considered a new feature.
+
+  example:
+  ```yaml
+  enhancements:
+    - |
+      Windows: Add PDH data to flare.
+  ```
+
+- `issues`: describe known issues or limitation of the agent.
+
+  example:
+  ```yaml
+  issues:
+    - |
+      Kubernetes 1.3 & OpenShift 3.3 are currently not fully supported: docker
+      and kubelet integrations work OK, but apiserver communication (event
+      collection, `kube_service` tagging) is not implemented
+  ```
+
+- `upgrade`: List action to take or limitation that could arise upon upgrading the agent.
+
+  example:
+  ```yaml
+  upgrade:
+    - |
+      If you run a Nomad agent older than 0.6.0, the `nomad_group`
+      tag will be absent until you upgrade your orchestrator.
+  ```
+
+- `deprecations`: List deprecation notes here.
+
+  example:
+  ```yaml
+  deprecations:
+  - |
+    Changed the attribute name to enable log collection from YAML configuration
+    file from "log_enabled" to "logs_enabled", "log_enabled" is still
+    supported.
+  ```
+
+- `security`: List security fixes, issues, warning or related topics here.
```

Comment:
still not very clear to me what this means :/ Is this section for fixed security issues or known security issues? (if we want to have sections for both, then I think we should have 2 different sections)

Reply:
I think it's about security related stuff: either security issue or fix, but also new behavior that may improve it ...

I think one section is fine, we almost never use it and it's more a way for user to quickly see what is important.